server/routes/movie.ts

```diff
@@ -118,7 +118,7 @@ movieRoutes.get('/:id/similar', async (req, res, next) => {
   }
 });
 
-movieRoutes.get('/:id/ratings', async (req, res, next) => {
+movieRoutes.get('/:id/ratingscombined', async (req, res, next) => {
```

Comment:
This is still a breaking change. We can't get rid of an endpoint either. Just make a new one entirely.

Reply:
@sct reinstated the previous endpoint